src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -151,39 +166,48 @@ export const OnLinkedIcons: Story = {
   render: () => (
     <Tooltip content="This is the tooltip text">
       <Link type="action" href="#passed-in-link">
-        <Icon decorative id="icon-id" name="search" size="xxlarge" />
+        <Icon decorative id="icon-id" name="search" size="large" />
       </Link>
     </Tooltip>
   ),
 };
 
-export const UsingIconContent: Story = {
+export const IconAndImageContent: Story = {
```

Comment:
See note above about adding a note that this should not be done.

Reply:
Oops, I see this here now!